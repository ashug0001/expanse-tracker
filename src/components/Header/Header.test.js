import React from "react";
import { shallow } from "enzyme";

import Header from "./Header";

describe("Header", () => {
  it("should render Header", () => {
    const wrapper = shallow(<Header title="Expanse Tracker" />);
  });

  it("should render initial layout", () => {
    const header = shallow(<Header title="Expanse Tracker" />);

    expect(header.getElement()).toMatchSnapshot();
  });
});
