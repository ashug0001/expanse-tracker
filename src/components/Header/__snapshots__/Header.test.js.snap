// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render initial layout 1`] = `
<h2>
  Expanse Tracker
</h2>
`;
